cayennelpp/lpp_data.py

```diff
@@ -1,6 +1,11 @@
 from .lpp_type import get_lpp_type
 
-import logging
+try:
+    import logging
+except ModuleNotFoundError:
```

Comment:
I think this should be `except ImportError:` there is no `ModuleNotFoundError` by default, that should also fix the flake8 issue(s) -> replace here and in the other 2 files